omnibus/config/software/more-ruby-cleanup.rb

```diff
@@ -27,54 +27,64 @@
 dependency "ruby"
 
 build do
+  block "remove any .gitkeep files" do
+    Dir.glob("#{install_dir}/**/{.gitkeep,.keep}").each do |f|
+      puts "Deleting #{f}"
+      File.delete(f)
+    end
```

Comment:
Wanna add a check to look for and delete empty folders after doing this?

Reply:
some of them are for specs so I wanted to keep them

Reply:
But a bunch are built into omnibus like bin, license, embedded/bin